scripts/promote-cherry-picks.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * Promotes cherry-picked release in related channels.
+ */
+
+import yargs from 'yargs/yargs';
```

Comment:
TODO: use yargs instead of minimist so we can require args, i.e. `string` instead of `string | null`